src/client/src/containers/DependencyInfo/index.tsx

```diff
@@ -119,9 +119,17 @@ class DependencyInfo extends React.Component<Props> {
       ? getSvg.getGreenCheckSvg()
       : getSvg.getWarningSvg();
 
+    const keyDownHandler = (event: React.KeyboardEvent<HTMLDivElement>) => {
+      if (event.keyCode === 13 || event.keyCode === 32) {
+        openPrivacyModal(dependency);
+      }
+    };
+
     return (
       <div
         role="button"
+        tabIndex={0}
```

Comment:
Just out of curiosity, what does `tabIndex={0}` mean?

Reply:
Good question, it allows you to navigate using tab to the element. The number defines the order of navigation (what is focused first)

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex